packages/core/contracts/optimistic-asserter/implementation/escalation-manager/BaseEscalationManager.sol

```diff
@@ -1,16 +1,36 @@
 pragma solidity 0.8.16;
 
+import "@openzeppelin/contracts/access/Ownable.sol";
 import "../../interfaces/EscalationManagerInterface.sol";
+import "../../interfaces/OptimisticAsserterInterface.sol";
 
 /**
  * @title BaseEscalationManager
  * @notice Base contract for escalation managers. This contract is responsible for managing the escalation policy for
  * assertions. This base implementation simply exposes the required interface and provides a default implementation
  * (returning default values or doing nothing).
  */
-contract BaseEscalationManager is EscalationManagerInterface {
+contract BaseEscalationManager is EscalationManagerInterface, Ownable {
+    OptimisticAsserterInterface public immutable optimisticAsserter;
+
     event PriceRequestAdded(bytes32 indexed identifier, uint256 time, bytes ancillaryData);
 
+    /**
+     * @notice Reverts unless the configured optimistic asserter is the caller.
+     */
+    modifier onlyOptimisticAsserter() {
+        require(msg.sender == address(optimisticAsserter), "Caller must be the optimistic asserter");
```

Comment:
Consider shorter message to fit 32 bytes slot, e.g. "Not authorized" or "Not the optimistic asserter"

Reply:
fixed